### ISSUE TITLE
How to generate **libMyTtrEngine-trt721.so**?

### ISSUE THREAD

**Issue by BrownOfSummer** [Wulingtian/RepVGG_TensorRT_int8#2]
How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?

**Comment by yang-gis** [comment]
> How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?

The authors did not want to open source this important code.If everyone is in this open source spirit, research will not start.

**Comment by yang-gis** [comment]
> How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?

The author's lack of open source makes it difficult to learn important things and to use them under Windows.

**Comment by yang-gis** [comment]
> How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?

Of course, the open source spirit of some Chinese people is very good, such as Enazoe, Wang-xinyu, etc.Their TensorTRT project is also very good, all open source for everyone to use and learn.

**Comment by Wulingtian** [comment]
> > How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?
> 
> Of course, the open source spirit of some Chinese people is very good, such as Enazoe, Wang-xinyu, etc.Their TensorTRT project is also very good, all open source for everyone to use and learn.

大佬，不用一下发这么多帖子吧，我先整理一下代码先，然后开源

**Comment by yang-gis** [comment]
> > > How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?
> > 
> > 
> > Of course, the open source spirit of some Chinese people is very good, such as Enazoe, Wang-xinyu, etc.Their TensorTRT project is also very good, all open source for everyone to use and learn.
> 
> 大佬，不用一下发这么多帖子吧，我先整理一下代码先，然后开源

First of all thank the authors engineering (although the important process is not open source code), then I know that many Chinese are so some things don't come easy makes its reluctance to easily open source these so-called important code, but if we don't open source we will never progress, our study do not rise forever, westerners often do is good.Open source or not is a matter of personal freedom, after all, it's your work.

**Comment by Wulingtian** [comment]
> > > > How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?
> > > 
> > > 
> > > Of course, the open source spirit of some Chinese people is very good, such as Enazoe, Wang-xinyu, etc.Their TensorTRT project is also very good, all open source for everyone to use and learn.
> > 
> > 
> > 大佬，不用一下发这么多帖子吧，我先整理一下代码先，然后开源
> 
> First of all thank the authors engineering (although the important process is not open source code), then I know that many Chinese are so some things don't come easy makes its reluctance to easily open source these so-called important code, but if we don't open source we will never progress, our study do not rise forever, westerners often do is good.Open source or not is a matter of personal freedom, after all, it's your work.

文件太大，我就放到百度盘了(链接: https://pan.baidu.com/s/1sF8vZ1JyBvk5Z_IUBP3CgA  密码: qlgk)

操作步骤如下：
cd Generate_LibMyTtrEngine-trt721/src
vim CMakeLists.txt 设置cuda目录（例如:/usr/local/cuda-11.0/include）
make build
cd build
cmake ..
make
在Generate_LibMyTtrEngine-trt721/bin目录下生成libMyTtrEngine-trt721.so动态库

**Comment by shining-love** [comment]
> > > > > How to generate repvgg_tensorrt_int8/trt_engine_lib/libMyTtrEngine-trt721.so ?
> > > > 
> > > > 
> > > > Of course, the open source spirit of some Chinese people is very good, such as Enazoe, Wang-xinyu, etc.Their TensorTRT project is also very good, all open source for everyone to use and learn.
> > > 
> > > 
> > > 大佬，不用一下发这么多帖子吧，我先整理一下代码先，然后开源
> > 
> > 
> > First of all thank the authors engineering (although the important process is not open source code), then I know that many Chinese are so some things don't come easy makes its reluctance to easily open source these so-called important code, but if we don't open source we will never progress, our study do not rise forever, westerners often do is good.Open source or not is a matter of personal freedom, after all, it's your work.
> 
> 文件太大，我就放到百度盘了(链接: https://pan.baidu.com/s/1sF8vZ1JyBvk5Z_IUBP3CgA 密码: qlgk)
> 
> 操作步骤如下：
> cd Generate_LibMyTtrEngine-trt721/src
> vim CMakeLists.txt 设置cuda目录（例如:/usr/local/cuda-11.0/include）
> make build
> cd build
> cmake ..
> make
> 在Generate_LibMyTtrEngine-trt721/bin目录下生成libMyTtrEngine-trt721.so动态库

thanks for your great work！You are very kind and have a great open source spirit！
